### ISSUE TITLE
[1.5.7] installation warnings

### ISSUE THREAD

**Issue by Cyrille37** [ether/etherpad-lite#2967]
Hi

While installing etherpad-lite 1.5.7 on node.js 0.12 :
```
npm WARN package.json ep_etherpad-lite@1.5.7 No license field.
npm WARN deprecated ueberDB@0.2.15: Package
npm WARN deprecated graceful-fs@3.0.6: graceful-fs v3.0.0 and before will fail on node releases >= v7.0. Please update to graceful-fs@^4.0.0 as soon as possible. Use 'npm ls graceful-fs' to find it in the tree.
npm WARN deprecated native-or-bluebird@1.1.2: 'native-or-bluebird' is deprecated. Please use 'any-promise' instead.
npm WARN deprecated jade@0.26.3: Jade has been renamed to pug, please install the latest version of pug instead of jade
npm WARN deprecated graceful-fs@2.0.3: graceful-fs v3.0.0 and before will fail on node releases >= v7.0. Please update to graceful-fs@^4.0.0 as soon as possible. Use 'npm ls graceful-fs' to find it in the tree.
npm WARN deprecated lodash@1.3.1: lodash@<3.0.0 is no longer maintained. Upgrade to lodash@^4.0.0.
```

**Comment by lpagliari** [comment]
There's already an issue opened for that (see #2954)

**Comment by pmontrasio** [comment]
I'm using node v6.9.2.
 
Jade is a dependency of mocha

```
$ npm ls jade
/home/me/etherpad/etherpad-lite
└─┬ ep_etherpad-lite@1.6.1 -> /home/me/etherpad/etherpad-lite/src
  └─┬ mocha@2.4.5
    └── jade@0.26.3 
```

The version of graceful-fs that triggers the warning is also a dependency of mocha. There are other newer versions of it used by other modules.

```
 $ npm ls graceful-fs
/home/me/etherpad/etherpad-lite
└─┬ ep_etherpad-lite@1.6.1 -> /home/me/etherpad/etherpad-lite/src
  ├── graceful-fs@4.1.3 
  ├─┬ mocha@2.4.5
  │ └─┬ glob@3.2.3
  │   └── graceful-fs@2.0.3 
  └─┬ npm@4.0.2
    └── graceful-fs@4.1.10 
```

The other problems have been fixed as of 1.6.1.
